catalog/resource_catalog_workspace_binding.go

```diff
@@ -2,66 +2,102 @@ package catalog
 
 import (
 	"context"
-	"strconv"
+	"fmt"
 
 	"github.com/databricks/databricks-sdk-go/apierr"
 	"github.com/databricks/databricks-sdk-go/service/catalog"
 	"github.com/databricks/terraform-provider-databricks/common"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
+var getSecurableName = func(d *schema.ResourceData) string {
+	securableName, ok := d.GetOk("catalog_name")
+	if !ok {
+		securableName = d.Get("securable_name")
```

Comment:
Can we do the reverse to prefer the new API?
```suggestion
	securableName, ok := d.GetOk("securable_name")
	if !ok {
		securableName = d.Get("catalog_name")
```